src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/Message.vue

```diff
@@ -28,6 +28,17 @@
             <span v-html="message.from == 'Registry' ? 'From ECE Registry' : 'PortalUser' ? 'You Replied' : ''"></span>
             <div class="mt-3" v-html="formatDate(String(message.notifiedOn), 'LLL d, yyyy t')"></div>
             <div class="mt-6" v-html="message.text"></div>
+            <div v-if="message.documents!.length > 0" class="mt-6">
+              <p>
+                <v-icon>mdi-paperclip</v-icon>
+                Attachments
+              </p>
+              <div v-for="(file, fileIndex) in message.documents" :key="fileIndex" class="mt-3">
+                <DownloadFileLink :get-file-function="() => getCommunicationFile(message.id || '', file.id || '')">
+                  <div>{{ `${file.name} (${file.size!.replace(/\s+/g, "")})` }}</div>
```

Comment:
I think we can use the already available Functions.humanFileSize(file.size!) instead {file.size!.replace(/\s+/g, "")} here and it will take care of the rest..